MIMWebClient/Core/Events/LoadRoom.cs

```diff
@@ -554,12 +551,9 @@ public static void ReturnRoom(Player player, Room room, string commandOptions =
                             {
                                 HubContext.SendToClient("You look into the " + playerItemDescription.name + " and see:", player.HubGuid);
 
-                                foreach (var containerItem in playerItemDescription.containerItems)
+                                foreach (var containerItem in playerItemDescription.containerItems.List())

```

Comment:
Pass true in the itemcontainer list so we get the article showing before the item.